fix(mobile): updates to hybrid mobile docs (#15689)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx

          * [Get a current session's ID](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id)
          * [Set a custom user ID to associate with events and attributes](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-custom-user-id)

> Learn about which would be the best method to use and why, see [Report mobile monitoring custom events and attributes](/docs/data-apis/custom-data/custom-events/report-mobile-monitoring-custom-events-attributes/).

Author: ```suggestion
To determine which method to use, see [Report mobile monitoring custom events and attributes](/docs/data-apis/custom-data/custom-events/report-mobile-monitoring-custom-events-attributes/).
```
